stack-deps.nix

```diff
@@ -28,7 +28,7 @@ let
         rev = "4067ad96b125942545dbdec8c1a89f1e1b65d013";
         sha256 = "1i9dlhw0xk1viglyhail9fb36v1awrypps8jmhrkz8k1bhx98ci3";
       };
-      cargoSha256 = "1373rpy0fi3cpacv06x1cv4cv0brwdri2680ymdkq8w44syp20ym";
+      cargoSha256 = "0m85c49hvvxxv7jdipfcaydy4n8iw4h6myzv63v7qc0fxnp1vfm8";
```

Comment:
should this land from https://github.com/wireapp/wire-server/pull/949?  (no strong opinion)

Reply:
I'll wait for #949 to be merged, then we don't need this here anymore.